src/Query/QueryBuilder.php

```diff
@@ -777,18 +776,13 @@ public function delete($delete = null, $alias = null)
      *
      * @return $this This QueryBuilder instance.
      */
-    public function update($update = null, $alias = null)
```

Comment:
Apps that currently calls `$qb->update()` without parameters will break after your change.